docroot/modules/custom/mass_content/src/EntitySorter.php

```diff
@@ -49,6 +42,18 @@ public function sortEntities(array &$entities, $sort) {
     }
   }
 
+  /**
+   * Sets a time format that avoid ambiguos date strings.
+   *
+   * To avoid potential ambiguity, it's best to use ISO 8601 (YYYY-MM-DD) dates
+   * or DateTime::createFromFormat() when possible.
+   *
+   * @see https://www.php.net/manual/en/function.strtotime.php
+   */
+  protected function formatDateValue($date, $convert_to_time = 1) {
```

Comment:
Should we add typehints to the method parameters, and a return type hint of `string`?

Why is `$convert_to_time` an integer and not a boolean?

Reply:
🤷🏼‍♂️ I don't know haha... Maybe I was tired and unconsciously decide that 1 and 0 were shorter compared to booleans. I will add the type hints for both parameters. Thanks!